src/lib/components/Swap/Summary.fixture.tsx

```diff
@@ -29,13 +35,18 @@ function Fixture() {
       [Field.OUTPUT]: UNI,
     })
     setInitialized(true)
-  })
+  }, [setState])
 
-  return initialized ? (
+  return initialized && trade ? (
     <Modal color="dialog">
-      <SummaryDialog onConfirm={() => void 0} />
+      <SummaryDialog onConfirm={() => void 0} trade={trade} />
     </Modal>
   ) : null
 }
 
-export default <Fixture />
+export default (
+  <>
+    <SwapInfoUpdater />
```

Comment:
Is the only way to have the trade meaningfully update to have the actual prices update through this updater? It would be good (for development/integration) if we could mock this.

Reply:
We could mock a trade?

Reply:
and remove the swap info hook

Reply:
low priority for now